packages/admin-ui/src/filters/__stories__/filters-searchable.stories.tsx

```diff
@@ -122,10 +123,8 @@ export function Async() {
   useEffect(() => {
     if (status === 'loading') return
 
-    combobox.value && !combobox.matches.length
-      ? setStatus('not-found')
-      : setStatus('ready')
-  }, [combobox.matches])
+    searchValue && !matches.length ? setStatus('not-found') : setStatus('ready')
```

Comment:
Maybe here would fit better an `if statement` or use the ternary expression with the status value. 
```suggestion
    const newStatus = searchValue && !matches.length ? 'not-found' : 'ready'
   setStatus(newStatus)
```